### ISSUE TITLE
Unable to load dynamic library '/usr/lib/php/20210902/event.so'

### ISSUE THREAD

**Issue by andreilutski** [shivammathur/setup-php#722]
**Describe the bug**

Installing `event` extension throws an error:

``
PHP Warning:  PHP Startup: Unable to load dynamic library '/usr/lib/php/20210902/event.so' (tried: /usr/lib/php/20210902/event.so (/usr/lib/php/20210902/event.so: undefined symbol: socket_ce), /usr/lib/php/20210902//usr/lib/php/20210902/event.so.so (/usr/lib/php/20210902//usr/lib/php/20210902/event.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
``

Maybe something going on here with this doubling? `/usr/lib/php/20210902//usr/lib/php/20210902/event.so.so (/usr/lib/php/20210902//usr/lib/php/20210902/event.so.so: cannot open shared object file: No such file or directory)`

**Version**
- [ ] I have checked releases, and the bug exists in the latest patch version of `v1` or `v2`.
- [x] `v2`
- [ ] `v1`

**Runners**

- [x] GitHub Hosted
- [ ] Self Hosted

**Operating systems**

Ubuntu 22.04

**PHP versions**

8.1.17

**To Reproduce**

```
- name: Setup PHP
  uses: shivammathur/setup-php@v2
  with:
    php-version: 8.1
      extensions: calendar, event, exif, gd, imagick, intl, mysqli, soap, sockets, opcache, zip
      tools: composer
```

**Expected behavior**

The `event` extension to be installed and enabled.

**Screenshots/Logs**

<img width="1097" alt="Screenshot 2023-04-18 at 17 08 41" src="https://user-images.githubusercontent.com/50401123/232803802-5e9ee33c-5dc6-402f-8106-c3aa37b61320.png">

**Additional context**

Just a plain PHP setup to run `paratest`.

**Are you willing to submit a PR?**

If I knew how to fix it.

**Comment by shivammathur** [comment]
@andreilutski 
Fixed in 19323ea920b7e413a7103da46df1061cb4a4f9a4. Will be in the next release.

**Comment by shivammathur** [comment]
Released in [`2.25.1`](https://github.com/shivammathur/setup-php/releases/tag/2.25.1)
